File: 프로그래머스/lv0/120890. 가까운 수/가까운 수.py
def solution(array, n):
    answer = []
    array.sort()
    for i in array:
        answer.append(abs(n-i))
    box = [array[answer.index(min(answer))]]
    if len(box)>1:
        return min(box)
    else:
        return box[0]

File: 프로그래머스/lv0/120890. 가까운 수/가까운 수.js
function solution(array, n) {
    var arr = [];
    array.sort((a,b)=>a-b);
    //차잇값을 배열에 넣기
    for (let a of array){
        arr.push(Math.abs(a-n));
    }
    //차잇값 가장 적은 첫번째 값 return 
    return array[arr.indexOf(Math.min(...arr))];
}